src/uct/ib/rc/accel/rc_mlx5.h

```diff
@@ -27,13 +27,17 @@ typedef struct uct_rc_mlx5_ep {
     struct ibv_qp    *tm_qp;
 } uct_rc_mlx5_ep_t;
 
+struct uct_ep_addr { int dummy; };
+
 typedef struct uct_rc_mlx5_ep_address {
-    uct_ib_uint24_t  qp_num;
     /* For RNDV TM enabling 2 QPs should be created, one is for sending WRs and
      * another one for HW (device will use it for RDMA reads and sending RNDV
      * Complete messages). */
-    uct_ib_uint24_t  tm_qp_num;
-    uint8_t          atomic_mr_id;
+    uct_ib_uint24_t     send_qp_num;
```

Comment:
maybe move send_qp_num to the end of the structure? could it help avoiding this union?